src/sagemaker_xgboost_container/constants/xgb_constants.py

```diff
@@ -11,6 +11,8 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
+XGB_VERSION = "1.5.2"
```

Comment:
Instead of hardcoding a version, consider using something like this https://stackoverflow.com/a/32965521

Reply:
will update